### ISSUE TITLE
Publishing to Nextcloud App Store?

### ISSUE THREAD

**Issue by e-alfred** [BatPio/PhotoMap#9]
Hello,

this app looks very interesting and would be very good fit for the Nextcloud App Store. As long as the Maps app is still in development, this App could bridge the gap and bring yet another feature to Nextcloud users until the Maps app is finished.

**Comment by BatPio** [comment]
Hi,

Sorry for the late reply, I just saw your message. Jancborchardt convinced me to work together on single maps app, and I decided not to publish my app. Today I merged my changes to maps/rework branch.

**Comment by rompe** [comment]
A year later, still no trace of the official maps app in the store. Maybe this one could at least fill the gap until nextcloud/maps#4 gets published somewhere?

**Comment by e-alfred** [comment]
Yes, the maps app is pretty much dead at this point. Maybe @BatPio can release it to the Nextcloud App store anyway?

**Comment by e-alfred** [comment]
Yes, the maps app is pretty much dead at this point. Maybe @BatPio can release this awesome app to the Nextcloud App store anyway?

**Comment by poVoq** [comment]
Is it even compatible with NC14?

**Comment by BatPio** [comment]
Hi, I stoped using Nextcloud because of [bug](https://github.com/nextcloud/server/issues/1732), it was really painfull to generate thumbails. I moved to alternative cloud software. Now I do not have Nextcloud instance to check if my app works with Nextcloud 14, sorry. Also, during porting my code to nextcloud/maps I made many performance improvements, so I recommend to continue working on maps app.
